docs/release_notes/release_notes.md

```diff
@@ -97,6 +88,5 @@ has_children: true
 ## Upgrade Guide from 2.3.0 to 3.0.0
 
 * Overview Dashboard
-    * We are no longer showing the alerts on the Overview page which did not trigger by default.
-    * Users can select a checkbox called "Show alerts that did not trigger" to see them.
-        * <TODO-Mahir> - something to be implemented.
+    * By default, non-triggered alerts will remain hidden. 
+    * Users can still view every alert by unchecking the **Hide Not Triggered Alerts** filter. 
```

Comment:
@mahirchavda  - add note on user to configure emails on both digest email alert and generic critical alert configuration along with the link of that doc page.